server/publicapi/features/returning_errors.feature

```diff
@@ -2,16 +2,29 @@ Feature: Returning error on invalid request parameters
 
     Scenario: Sending an unknown parameter when retrieving items
         When we get "/items?parameter_x=foo"
-        Then we get error 422
+        Then we get error 10001
```

Comment:
I think we decided for regular HTTP codes because it provides better semantics, and because the content API is treated as an independent service that does not share much with the "internal" API that serves the Superdesk client. These two have different goals and do not have to share the error codes and other features, @hlmnrmr can elaborate on that.

Anyway, thanks for merging, I'll rebase my e2e API tests branch now.